.docker/docker_run_git.sh

```diff
@@ -30,6 +30,11 @@ fi
 # From now, stop at error
 set -e
 
+if [ $PS_DEV_MODE -ne 1 ]; then
+  echo "\n* Disabling DEV mode ...";
+  sed -ie "s/define('_PS_MODE_DEV_', true);/define('_PS_MODE_DEV_',\ false);/g" /var/www/html/config/defines.inc.php
```

Comment:
@boubkerbribri just wondering, why did you use `-ie` flag which generates `defines.inc.phpe` file? Is this supposed to be a backup? What's the point of that?

Reply:
I think it's an error that slipped away here

Reply:
I'm making a PR for the config so I'll fix that too :)